operations/helm/charts/grafana-agent/values.yaml

```diff
@@ -252,13 +254,16 @@ serviceMonitor:
 
   # -- RelabelConfigs to apply to samples before scraping
   # ref: https://github.com/prometheus-operator/prometheus-operator/blob/main/Documentation/api.md#relabelconfig
-  relabelings: []
   # - sourceLabels: [__meta_kubernetes_pod_node_name]
   #   separator: ;
   #   regex: ^(.*)$
   #   targetLabel: nodename
   #   replacement: $1
   #   action: replace
+  relabelings: []
```

Comment:
nit: can we move the example below the relabelings so that it doesn't get rendered weirdly on the README?